src/components/forms/pool_actions/WithdrawForm.vue

```diff
@@ -59,7 +59,10 @@
               <span class="break-words">
                 {{ amountUSD(i) === 0 ? '-' : fNum(amountUSD(i), 'usd') }}
               </span>
-              <span class="text-xs text-gray-400">
+              <span
+                v-if="pool.poolType !== 'Stable'"
```

Comment:
Would be great to have this as an enum/constant

Reply:
Agreed: https://github.com/balancer-labs/frontend-v2/pull/570/commits/fde9bfe897adbb9a83796085b0db32c1eaf8717b